src/site/_includes/components/macro-options-table/macro.njk

```diff
@@ -0,0 +1,38 @@
+{% macro guidanceMacroOptionsTable(params) -%}
+
+<table class="govuk-table guidance-options__table" id="options-{{ params.exampleId }}--{{ params.tableId }}">
+  <caption class="govuk-table__caption govuk-heading-m {% if params.tableId == 'primary' %} govuk-visually-hidden{% endif %}">{{ params.name }}</caption>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header guidance-options__limit-table-cell" scope="col">Name</th>
+      <th class="govuk-table__header guidance-options__limit-table-cell" scope="col">Type</th>
+      <th class="govuk-table__header" scope="col">Description</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body guidance-options-table">
+    {% for option in params.table -%}
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header" scope="row">{{option.name}}</th>
+        <td class="govuk-table__cell">{{option.type}}</td>
+        <td class="govuk-table__cell guidance-macro-option-description">
+          {% if option.required === true %}
+            <strong>Required.</strong>
+          {% endif %}
+          {{ option.description | markdown }}
```

Comment:
It's a shame you can't use 
https://www.11ty.dev/docs/plugins/render/#rendertemplate

Reply:
Oh interesting, didn't see that. Seems to completely break in the macro though (even with static content) – just saw that's async so it'll be that :weary: